app/src/main/java/org/p2p/wallet/common/feature_toggles/toggles/remote/SwapRoutesRefreshFeatureToggle.kt

```diff
@@ -0,0 +1,13 @@
+package org.p2p.wallet.common.feature_toggles.toggles.remote
+
+import org.p2p.wallet.common.feature_toggles.remote_config.RemoteConfigValuesProvider
+
+private const val DEFAULT_INTERVAL_IN_MILLIS = 20_000L
+
+class SwapRoutesRefreshFeatureToggle(
+    valuesProvider: RemoteConfigValuesProvider
+) : LongFeatureToggle(valuesProvider) {
+    override val featureKey: String = "swap_route_refresh"
+    override val featureDescription: String = "The interval for refreshing routes"
+    override val defaultValue: Long = DEFAULT_INTERVAL_IN_MILLIS
```

Comment:
what if you add here valueDuration: Duration and use it like 
val valueDuration:Duration = value.milliseconds

so we won't guess what time unit is inside the actual value